packages/integration/src/virtualFile.ts

```diff
@@ -1,12 +1,18 @@
+import zlib from 'zlib';
+
 export function getSourceFromVirtualCssFile(id: string) {
   const match = id.match(/^(?<fileName>.*)\?source=(?<source>.*)$/) ?? [];
 
   if (!match || !match.groups) {
     throw new Error('No source in vanilla CSS file');
   }
 
+  const decompressedSource = zlib.gunzipSync(
```

Comment:
Not a huge deal but maybe we could create some shared `compress` and `decompress` functions just to make sure all the places dealing with compression are using the same base64 and compression logic? You could export it from the integration package and use it in the virtual loader code.